ctapipe/instrument/camera.py

```diff
@@ -203,6 +203,21 @@ def _calc_pixel_area(pix_x, pix_y, pix_type):
 
         return np.ones(pix_x.shape) * area
 
+    @lazyproperty
+    def all_pixels_kdtree(self):
```

Comment:
is there a case where this is not for all pixels? Otherwise, perhaps just name this property `kdtree`

Reply:
Renamed it to simply kdtree in f1a7b92d. If finally satisfied with this PR, maybe have a look at already existing _find_neighbor_pixels method() to work with the same kdtree (seems to also use all pixels, but slightly differently implemented).